.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: CI robot-browserstack sample
+on:
+  pull_request:
```

Comment:
Can we explore comment option?

Reply:
https://github.com/browserstack/BStackAutomation/blob/1fa3e8eacbba942a8b7a78910b136ab9551679cb/.github/workflows/nw_linter.yml

Reply:
yeah, will do